src/utils/math_utils.py

```diff
@@ -53,6 +53,10 @@ def compute_bounding_box(points):
     return min_x, max_y, max_x, min_y
 
 
+def euclid_distance(v1, v2):
+    return np.sqrt(np.sum((v1 - v2) ** 2))
```

Comment:
Hebben we hier een aparte method voor nodig? Dit kan ook gewoon met
`np.linalg.norm(v1 - v2)`